package.json

```diff
@@ -8,7 +8,9 @@
     "start": "next start"
   },
   "dependencies": {
+    "dotenv": "^8.2.0",
```

Comment:
You don't need this package in this project. Check this out: https://nextjs.org/docs/api-reference/next.config.js/environment-variables